tests/BaseTestCase.php

```diff
@@ -44,11 +57,13 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('mailviewer.allowed_environments', ['local', 'staging', 'testing']);
         $app['config']->set('mailviewer.middlewares', []);
 
-        $app->singleton(EloquentFactory::class, function ($app) {
-            $faker = $app->make(\Faker\Generator::class);
-            $factories_path = __DIR__ . '/Factories';
+        $app['config']->set('database.default', 'testbench');
+        $app['config']->set('database.connections.testbench', [
+           'driver' => 'sqlite',
+           'database' => ':memory:',
+           'prefix' => '',
+        ]);
 
-            return EloquentFactory::construct($faker, $factories_path);
-        });
+        $app['config']->set('app.debug', env('APP_DEBUG', true));
```

Comment:
Out of curiosity, any specific reason for setting this config? 😄

Reply:
When there is an error somewhere in the code the tests that fail dump the generated error page (because of the `assertSee`). Without this setting it is just the standard error page, with this setting enabled you can check the dumped html for the actual error so just helps debugging.